MangoPay/PayOut.php

```diff
@@ -19,12 +33,24 @@ class PayOut extends Transaction
      */
     public $PaymentType;
 
+    /**
+     * One of PayOutPaymentDetails implementations, depending on $PaymentType
+     * @var object
+     */
+    public $PayoutPaymentDetails;
+
     /**
      * One of PayOutPaymentDetails implementations, depending on $PaymentType
      * @var object
      */
     public $MeanOfPaymentDetails;
 
+    /**
+     * Payout mode requested, default is 'STANDARD', Allowed values are
+     * 'STANDARD', 'INSTANT_PAYMENT', 'INSTANT_PAYMENT_ONLY'
+     * @var string
+     */
+    public $PayoutModeRequested;
```

Comment:
What is the reason for adding this to the PayOut class?

Reply:
This property was identified in both the documentation and PHP SDK. I verified its existence through debug logs and implemented it accordingly.